distributed/comm/tests/test_ws.py

```diff
@@ -121,7 +121,7 @@ async def test_roundtrip(c, s, a, b):
 
 @gen_cluster(client=True, scheduler_kwargs={"protocol": "ws://"})
 async def test_collections(c, s, a, b):
-    da = pytest.importorskip("dask.array")
+    da = pytest.importorskip("dask.array", exc_type=ImportError)
```

Comment:
this is a little odd. why is this necessary?

Reply:
It's either that or adding a `pytest.importorskip("numpy")`. Basically, we raise an ImportError when importing dask.array but don't have the required dependencies over here:

https://github.com/dask/dask/blob/ff2488aec44d641696e0b7aa41ed9e995c710705/dask/array/__init__.py#L266-L273

More context from the `pytest` docs:

> If the module can be imported but raises [ImportError](https://docs.python.org/3/library/exceptions.html#ImportError), pytest will issue a warning to the user, as often users expect the module not to be found (which would raise [ModuleNotFoundError](https://docs.python.org/3/library/exceptions.html#ModuleNotFoundError) instead).
>
> This warning can be suppressed by passing exc_type=ImportError explicitly.

(https://docs.pytest.org/en/stable/reference/reference.html#pytest-importorskip)

Reply:
I would do a 

pytest.importorskip("numpy")

this is what we do in dask/dask as well

Reply:
Works for me.